spring-cloud-huawei-governance/src/main/java/com/huaweicloud/governance/adapters/web/RestTemplateUtils.java

```diff
@@ -58,11 +58,6 @@ public String instanceId() {
 
       @Override
       public String serviceName() {
-        RetryContext retryContext = InvocationContextHolder.getOrCreateInvocationContext()
```

Comment:
Can you add a test case for this issue? And also include Feign senario.

Reply:
OK I,ll do it